db/structure.sql

```diff
@@ -2,13 +2,29 @@
 /*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
 /*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
 /*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
-/*!40101 SET NAMES utf8mb4 */;
+/*!40101 SET NAMES utf8 */;
```

Comment:
This is from production so it must be right, but... is this right?

Reply:
I think this is just the overall default. Most tables have explicit overrides (`CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci`) which haven't changed.